packages/framework/ai-collab/api-report/ai-collab.alpha.api.md

```diff
@@ -4,6 +4,50 @@
 
 ```ts
 
+// @alpha
+export function aiCollab<TSchema extends ImplicitFieldSchema>(options: AiCollabOptions<TSchema>): Promise<AiCollabSuccessResponse | AiCollabErrorResponse>;
+
+// @alpha
+export interface AiCollabErrorResponse {
+    // (undocumented)
```

Comment:
Please make sure to document the undocumented members before merging 🙂

Reply:
Added the remaining jsdoc for aiCollabApi members in commits a79e33c651f4084ad3e7f7c04689ef61d95f4cd7 and b6755728ce844e69b98b9dd01ebb510111b187d0